src/LanguageServer.Engine/Documents/MasterProjectDocument.cs

```diff
@@ -67,15 +68,21 @@ protected override void Dispose(bool disposing)
         /// <summary>
         ///     Add a sub-project.
         /// </summary>
-        /// <param name="subProjectDocument">
+        /// <param name="documentUri">
         ///     The sub-project.
```

Comment:
This is why I don't like oversaturation of doc comments. This one is now obsolete and needs updating while IMO `Uri documentUri` is descriptive enough on its own and don't need additional commenting. Please just update the doc comment here, if we want to get rid of unnecessary noice we should do it in 1 pass as part of #91

Reply:
Yeah fair call, I’ll update it to match for now but we can sort it out properly later (e.g. #91).